src/query_ctx.c

```diff
@@ -76,7 +76,6 @@ void QueryCtx_SetLastWriter(OpBase *last_writer) {
 
 AST *QueryCtx_GetAST(void) {
 	QueryCtx *ctx = _QueryCtx_GetCtx();
-	ASSERT(ctx->query_data.ast);
```

Comment:
This assertion is invalid for certain compile-time errors, like non-string key values in a map.